IdentityCore/src/workplacejoin/MSIDPkeyAuthHelper.m

```diff
@@ -72,7 +72,9 @@ + (nullable NSString *)createDeviceAuthResponse:(nonnull NSString*)authorization
 {
     MSIDRegistrationInformation *info =
     [MSIDWorkPlaceJoinUtil getRegistrationInformation:context urlChallenge:nil];
-    NSString *pKeyAuthHeader = @"";
+    NSString *authToken = @"";
+    NSString *challengeContext = challengeData ? [challengeData valueForKey:@"Context"] : @"";
```

Comment:
nit: should we also make sure challengeData is dictionary unless the code above already ensures that?